Rakefile

```diff
@@ -10,7 +10,7 @@ task :setup => ["bundler:checkout"] do
   gemspec = Gem::Specification.load(File.expand_path("../rubygems-update.gemspec", __FILE__))
 
   gemspec.dependencies.each do |dep|
-    sh "gem install '#{dep.name}:#{dep.requirement.to_s}' --conservative --no-document"
+    sh "gem install '#{dep.name}:#{dep.requirement.to_s}' --conservative --no-document --force"
```

Comment:
Not sure what's the reason for this, but I needed this when trying out the [`ronn-ng`](https://github.com/apjanke/ronn-ng) gem. Since I already had the [`ronn`](https://github.com/rtomayko/ronn) gem installed, without `--force` the `ronn`  executable (provided by both gems) wouldn't be updated. So, :+1:.